lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -17,38 +17,13 @@ def define
       multiple_choice :going_or_already_abroad? do
         option :going_abroad
         option :already_abroad
-        save_input_as :going_or_already_abroad
 
-        on_response do
+        on_response do |response|
           self.calculator = uk_benefits_abroad_calculator
```

Comment:
```suggestion
          self.calculator = Calculators::UkBenefitsAbroadCalculator.new
```
For consistency I think we want instantiate the calculator here. This also every request receives a new calculator - and prevent issues of state persisting between different requests.

Reply:
`uk_benefits_abroad_calculator` is used 8 times within the flow. It used in blocks like this (lines 288-290):
```ruby
uk_benefits_abroad_calculator.state_benefits.each_key do |benefit|
  option benefit
end
```

Reply:
In these cases could we refer to the class constant rather than a instance method (there's also no reason for this to be an instance method) 
```ruby
Calculators::UkBenefitsAbroadCalculator::STATE_BENEFITS.each_key do |benefit|
  option benefit
end
```